index.ts

```diff
@@ -1,54 +1,31 @@
 import { multiUrlGenerator } from './lib/multiUrlGenerator';
-import readline from 'readline';
+import fetch from 'node-fetch';
 
-const clientUI = readline.createInterface({
```

Comment:
I removed the console client UI because this will not be the final use case of this app (we'll feed the app with a json config file), something like
```
npm run humblescrapper --file urlComponents.json
```

Reply:
👍